src/layouts/base/MainLayout.tsx

```diff
@@ -2,7 +2,7 @@ import { Header } from "src/components/header/Header"
 
 export const MainLayout = () => {
     return (
-        <div>
+        <div className={"wrapper"}>
```

Comment:
```suggestion
        <div className={"main-layout"}>
```

Корневой див называть также, как называется компонент

Reply:
исправлено